superset/db_engine_specs/mysql.py

```diff
@@ -93,6 +104,21 @@ class MySQLEngineSpec(BaseEngineSpec):
 
     type_code_map: Dict[int, str] = {}  # loaded from get_datatype only if needed
 
+    custom_errors = {
+        INVALID_ACCESS_REGEX: (
+            __('Either the username "%(username)s" or password is incorrect'),
```

Comment:
"... or **the** password is incorrect." would be better?

Also, all messages should end in a period.